python/test/Ice/blobject/test.py

```diff
@@ -3,6 +3,6 @@
 # This test doesn't support running with IceSSL, the Router object in the client process uses
 # the client certificate and fails with "unsupported certificate purpose"
 
-from Util import TestSuite
+from Util import TestSuite  # type: ignore
```

Comment:
Should this have a TODO?

Reply:
Probably not, I doubt we'll want to add type hinting to the Util code.